src/components/List/List.jsx

```diff
@@ -11,6 +11,8 @@ const List = () => {
 
   const [value, loading, error] = useCollection(firebasePath);
 
+  const [filter, setFilter] = useState('');
```

Comment:
Nice job with this state controlling the input field and doing the filtering. Super nit-picky comment, but I think that we should name this after the input that it controls rather than the filtering that happens as a by-product of its existence. Maybe something like `inputValue`, `setInputValue`